doc/lua_api.txt

```diff
@@ -4117,6 +4248,14 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ Uses texture (string)
         playername = "singleplayer"
     --  ^ optional, if specified spawns particle only on the player's client
+        material_type_param = 0,
+    --  ^ optional, if specified spawns particle with
+    --  specified material type param, refer to "Particle blend" section
+        animation = {Animation definition},
+    --  ^ optional, note that particle and particle spawners have differences
+        glow = 0,
```

Comment:
@paramat also consider how we should check what type of glow to use. Should it be boolean or string enum.